change/@fluentui-babel-make-styles-e24db7df-1613-42dd-aa1a-a4ddb5b85b83.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
You should merge master to get rid of all these change files. They were removed recently by Sean.